contentwarehouse/snippets/create_document_sample.py

```diff
@@ -16,25 +16,28 @@
 
 # [START contentwarehouse_create_document]
 
+from typing import Optional
+
 from google.cloud import contentwarehouse_v1 as contentwarehouse
 
 
 def sample_create_document(
     project_number: str,
     location: str,
-    raw_doc_path: str,
-    mime_type: str,
+    raw_document_path: str,
+    raw_document_file_type: int,
     document_schema_id: str,
     user_id: str,
-    reference_id: str = "",
+    reference_id: Optional[str] = None,
 ) -> contentwarehouse.types.CreateDocumentResponse:
     """Creates a document.
 
     Args:
         project_number: Google Cloud project number.
         location: Google Cloud project location.
-        raw_doc_path: Raw document file in Cloud Storage path.
-        mime_type: Document format eg. PDF/DOCX etc.
+        raw_document_path: Raw document file in Cloud Storage path.
+        raw_document_file_type: Document file type,
+                                reference:https://cloud.google.com/python/docs/reference/contentwarehouse/latest/google.cloud.contentwarehouse_v1.types.RawDocumentFileType.
```

Comment:
```suggestion
        raw_document_file_type: Document file type
            https://cloud.google.com/document-warehouse/docs/reference/rest/v1/Document#rawdocumentfiletype
```

Reply:
I've changed it to use int instead of RawDocumentFileType object

Reply:
I'd recommend against that. You should use the raw document file type enum. You can also hardcode this value in for the sample.